_posts/release/2017-09-05-accumulo-2.0.0.md

```diff
@@ -18,6 +18,16 @@ always return `false` instead of `true`, even if the contents are equal.
 [ACCUMULO-3652] - Replaced string concatenation in log statements with slf4j
 where applicable. Removed tserver TLevel logging class.
 
+[ACCUMULO-4733] - Introduced a new configuration option,
```

Comment:
The `[link]` indicates a link in markdown.  You just need to add the URLs to the bottom of the page:

```
[ACCUMULO-4733]: https://issues.apache.org/jira/browse/ACCUMULO-4733
[ACCUMULO-4737]: https://issues.apache.org/jira/browse/ACCUMULO-4737
```